script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -195,16 +197,16 @@ I get the feeling that this boy definitely knows what I should be doing from her
 [sha:49e90d033c74fadaf73e8585782ac7707aa8d18c]{
 -- Page 686, Offset 26113.
 -- 「あの、代行？　わたしもですか？　っていうか、座って休んでいいですか？」
-"Um, Executor, sir? May I come as well? Or rather, can I sit down and rest?"
+"Um, Presbyter, sir? May I come as well? Or rather, can I sit down and rest?"
```

Comment:
I feel 'may' should be 'should' instead. It feels like she's trying to phrase it in a way that shifts the intent to Mario rather than herself.

Reply:
maybe even "shouldn't I" since she's feelin' left out